src/SelectSingle.vue

```diff
@@ -293,7 +300,7 @@
 				<span v-else-if="isPlaceholderShown" class="combo-placeholder">{{ placeholder }}</span>
 				<!-- @slot Display the currently selected option via custom template code -->
 				<slot v-else name="selectedOption" :option="value">
-					{{ value[labelField] }}
+					{{ this.prependLabel ? label + ': ' + value[labelField] : value[labelField] }}
```

Comment:
```suggestion
					{{ prependLabel ? label + ': ' + value[labelField] : value[labelField] }}
```

`this` isn't exposed to the template in Vue. Templates have access to the `computed`, `props` and `data` properties without using `this`.

Reply:
Fixed, thank you!

Reply:
Weirdly enough, I was not seeing any errors with my local storybook, I'm not sure why.